packages/dds/tree/src/feature-libraries/forest-summary/forestSummarizer.ts

```diff
@@ -34,24 +38,36 @@ import type {
 	SummaryElementStringifier,
 } from "../../shared-tree-core/index.js";
 import { idAllocatorFromMaxId } from "../../util/index.js";
-import { chunkFieldSingle, defaultChunkPolicy } from "../chunked-forest/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { chunkFieldSingle, defaultChunkPolicy } from "../chunked-forest/chunkTree.js";
 import type { FieldBatchCodec, FieldBatchEncodingContext } from "../chunked-forest/index.js";
 
 import { type ForestCodec, makeForestSummarizerCodec } from "./codec.js";
 import type { Format } from "./format.js";
+import { ForestIncrementalSummaryBuilder } from "./incrementalSummaryBuilder.js";
+import { TreeCompressionStrategyExtended } from "../treeCompressionUtils.js";
+
 /**
- * The storage key for the blob in the summary containing tree data
+ * The key for the tree that contains the overall forest's summary tree.
+ * This tree is added by the parent of the forest summarizer.
  */
-const treeBlobKey = "ForestTree";
+export const forestSummaryKey = "Forest";
+
+/**
+ * The key for the blob in the summary containing the forest's contents.
+ */
+export const forestSummaryContentKey = "ForestTree";
```

Comment:
This is ambiguous phrasing. It could either mean "the blob (which contains the forest's contents) in the summary" or a blob in a summary, where the summary contains the forest's contents.

Also as this is only used within this field as an implementation detail of ForestSummarizer, I think not exporting it would improve clarity around how it's used.
```suggestion
 * The key for the blob under ForestSummarizer's root.
 * This blob contains the ForestCodec's output.
 */
const forestSummaryContentKey = "ForestTree";
```